package.json

```diff
@@ -35,7 +35,9 @@
   "scripts":  {
     "test": "./node_modules/.bin/mocha test/unit -R spec -t 5000",
     "ci": "./node_modules/.bin/mocha test/unit test/integration/short -R mocha-multi",
-    "ci_unit": "./node_modules/.bin/mocha test/unit -R mocha-multi"
+    "ci_unit": "./node_modules/.bin/mocha test/unit -R mocha-multi",
+    "ci_smoke": "./node_modules/.bin/mocha test/unit -R mocha-multi; ./node_modules/.bin/mocha test/integration/short -g \"\\u007csmoke\\u007c\" -R mocha-multi",
+    "ci_smoke_win": "./node_modules/.bin/mocha test/unit -R mocha-multi & \"./node_modules/.bin/mocha\" test/integration/short -g \"\\u007csmoke\\u007c\" -R mocha-multi"
```

Comment:
This is a bit bizarre for windows to have a separate command like this, explaining all the nuances.

* `\u007c` is a vertical pipe, used instead of `|` because that throws windows off.
* `\\u007c` the backslash is escaped, otherwise it is interpreted as '|' before calling script.
* Double quotes are used because single quotes don't work on windows
* A separate command for windows is needed because `;` doesn't work to separate commands on command line.  `&&` could work, but if any of the unit tests fail, the smoke tests won't run (although maybe that's ok?).
* We run mocha twice because we don't want the smoke grep filter applied to the unit tests.
* In `ci_smoke_win` the second mocha call is called in escaped quotes, this is needed for some reason after the `&`.  However, it is not valid for the first mocha invocation.